client/components/forms/form-radio/style.scss

```diff
@@ -0,0 +1,45 @@
+.form-radio {
+	@extend %form-field;
+
+	clear: none;
+	cursor: pointer;
+	display: inline-block;
+	line-height: 0;
+	height: 16px;
+	margin: 2px 0 0;
+	float: left;
+	outline: 0;
+	padding: 0;
+	text-align: center;
+	vertical-align: middle;
+	width: 16px;
+	min-width: 16px;
+	appearance: none;
+
+	border-radius: 50%;
+	margin-right: 4px;
+	line-height: 10px;
+
+	&:checked::before {
+		float: left;
+		display: inline-block;
+		content: '\2022';
+		margin: 3px;
+		width: 8px;
+		height: 8px;
+		text-indent: -9999px;
+		background: var( --color-primary );
+		vertical-align: middle;
```

Comment:
Some of those declarations (for example `vertical-align: middle` seem to be unnecessary here as they're already declared above for `.form-radio`.